data/schema.yaml

```diff
@@ -106,42 +105,49 @@ components:
           type: string
           format: date-time
           example: 2017-12-21T17:32:28Z
-        MetaDescription:
+        metaDescription:
           type: boolean
           example: null
         model:
           type: object
           properties:
             blocks:
-              properties:
-                blockId:
-                  type: string
-                  example: 0f29a57f-3492-3c0d-af51-837208b657fb
-                type:
-                  type: string
-                  enum: [Headline, Text,altText, paragraph, Image, caption, Video, rawImage, rawVideo]
-                model:
-                  oneOf:
-                  - $ref: '#/components/schemas/Article/properties/model/properties/blocks'
-                  - $ref: '#/components/schemas/Text'
-                  - $ref: '#/components/schemas/Image'
-                  - $ref: '#/components/schemas/Headline'
-                  - $ref: '#/components/schemas/altText'
-                  - $ref: '#/components/schemas/Paragraph'
-                  - $ref: '#/components/schemas/Caption'
-                  - $ref: '#/components/schemas/Video'
-                  - $ref: '#/components/schemas/rawVideo'
-                  - $ref: '#/components/schemas/rawImage'
-                  - $ref: '#/components/schemas/Model'
-                  - $ref: '#/components/schemas/rawImageBlock'
-                  - $ref: '#/components/schemas/rawVideoBlock'
-
+              type: array
```

Comment:
is there a way to enforce we have to have 1 or more items inside of the blocks array